jscomp/others/js_json.ml

```diff
@@ -108,10 +108,12 @@ let decodeNull json : _ Js.null option =
 external parseExn : string -> t = "parse"
   [@@bs.val] [@@bs.scope "JSON"]
 
-external stringifyAny : 'a -> string option =
+external stringifyAnyExn : 'a -> string option =
```

Comment:
Wouldn't it be better to return just `string`?
`JSON.stringify` may return `undefined` but only when invalid JSON was given.

Reply:
I think, `stringify: t -> string` is fine, because t means that the valid JSON was given.
But, I'm afraid that it is better `stringifyAnyExn` has string option, because the invalid JSON can be given.